src/App.jsx

```diff
@@ -1,19 +1,16 @@
-import React from 'react';
 import Calculator from './components/Calculator';
 import Navigation from './components/Nav';
 
-class App extends React.PureComponent {
-  render() {
-    return (
-      <div>
-        <header><Navigation /></header>
-        <main className="main-container">
-          <Calculator />
-        </main>
-        <footer className="footer-container" />
-      </div>
-    );
-  }
+function App() {
+  return (
+    <div>
+      <header><Navigation /></header>
+      <main className="main-container">
+        <Calculator />
+      </main>
+      <footer className="footer-container" />
+    </div>
+  );
 }
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job by changing this to a function based. But I think it would be better if you use the ES6 syntax which means `the arrow function` for best practices. 

```JavaScript
const App = () => (
    // reference as a js object
      <div>
      ..... your code...
      </div>
);
```

Reply:
Thanks for the advice. Changed the syntax.